src/main/config/conversion/import/iso19139.ca.HNAP.FGP-to-iso19139.ca.HNAP.xsl

```diff
@@ -119,6 +119,29 @@
     </xsl:copy>
   </xsl:template>
 
+  <!--Add default LocalisedCharacterString to thumbnail -->
+  <xsl:template match="gmd:fileDescription">
+    <xsl:copy copy-namespaces="no">
+      <gmd:PT_FreeText>
+        <gmd:textGroup>
+          <xsl:choose>
+            <xsl:when test="$mainLanguage='eng'">
+              <gmd:LocalisedCharacterString locale="#fra">
+                <xsl:value-of select="./gco:CharacterString/text()"/>
+              </gmd:LocalisedCharacterString>
+            </xsl:when>
+            <xsl:otherwise>
+              <gmd:LocalisedCharacterString locale="#eng">
+                <xsl:value-of select="/gco:CharacterString"/>
+              </gmd:LocalisedCharacterString>
+            </xsl:otherwise>
+          </xsl:choose>
+        </gmd:textGroup>
+      </gmd:PT_FreeText>
+      <xsl:apply-templates select="node()|@*"/>
```

Comment:
This copies the `gco:CharacterString` after `gmd:PT_FreeText` causing a xsd error.

Another stuff is that the code doesn't check if the `gmd:PT_FreeText` is already defined with a value.

Reply:
Can you be more specific about what xsd error or provide a sample test data here? I can look for handle existing gmd:PT_FreeText issue.